Ginger/Ginger/UserControlsLib/ImageMakerLib/ImageMakerControl.xaml.cs

```diff
@@ -300,16 +300,16 @@ private void SetImage()
                 #region Execution Status Images
                 //############################## Execution Status Images:
                 case eImageType.Passed:
-                    SetAsFontAwesomeIcon(EFontAwesomeIcon.Regular_CheckCircle, (SolidColorBrush)FindResource("$PassedStatusColor"), 0, "Passed");
```

Comment:
Please verify all places where this imagecontrols are being used

Reply:
Yes, I have verified. There was only one changed to be made to resolve the issue.
I only had to change the image type from "Stop" to "Stopped".